client/src/services/Normalizer.js

```diff
@@ -0,0 +1,25 @@
+export const eventTypeMapper = eventType => ({
```

Comment:
call service files in camel case. And move this normalizer to eventType module, as it relates for that data